.github/pull_request_template.md

```diff
@@ -0,0 +1,15 @@
+<!--- Provide a general summary of your changes in the Title above -->
+
+## Changelist
+<!-- Bulleted list of changes made -->
+- 
+
+## Breaking Changelist
+<!-- Delete section if none, bulleted list of breaking changes -->
+- 
+
+## Issue number/link
+<!-- Link to related issues -->
```

Comment:
Should we also mention the github feature on the right hand side (see screenshot)? The text next to that feature is a bit confusing and should say "related issues" or something like that, but I like to make use of it because it's neat.

![image](https://github.com/astriaorg/astria/assets/701177/b117d549-88ea-4cb0-b897-f6ac418f6c4c)